src/display/editor/editor.js

```diff
@@ -213,7 +213,7 @@ class AnnotationEditor {
     this.div.setAttribute("data-editor-rotation", (360 - this.rotation) % 360);
     this.div.className = this.name;
     this.div.setAttribute("id", this.id);
-    this.div.tabIndex = 100;
+    this.div.tabIndex = 0;
```

Comment:
This wrapping div shouldn't have a tabindex. The contenteditable is already focusable and we should rely on that for focusability. Otherwise, we end up with a pointless (arguably broken) tab stop here.

Reply:
The `contenteditable` attribute is removed when the editor isn't in an editable state: for example when the user drags it to move it somewhere else on the page.
Anyway I'll remove the tab-index when the editor is in editing mode.
And there is too an "ink" editor in order to draw something on a pdf

Reply:
> The contenteditable attribute is removed when the editor isn't in an editable state: for example when the user drags it to move it somewhere else on the page.

Ah. It'll be hard for screen reader users to drag anyway, but regardless, I'm curious: what happens when a user drags the editor? Does it become non-editable altogether after that point or just while the user is holding down the mouse button?

Reply:
A text editor can changed in double-clicking on it (simple click makes it the current one).
The idea behind being able to move is, is to help to adjust the position when you want to fill a form.